### DISCUSSION TITLE
Use liquids nearby for fermenting vat

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#2644] [General]
A fermenting vat only accepts liquids inside your inventory, even if a wooden keg is nearby with a fermentable liquid. With an empty container, or a rubber hose, you should be able to transfer one to the other automatically, depending on distance and quantity.

This is mostly suggested as it confuses new players, but it would be a nice quality of life feature.

**Comment by Unrepentant-Atheist** [comment]
Yea, I agree. Same with the fermenting vat too. Can't transfer liquid from vat to empty keg nearby. Has to be a liquid container in inventory or dropped on the tile.

**Reply by yay855** [reply]
Yep, makes total sense to me. Especially useful for large-scale brewing operations, since the wooden keg is an easy large-volume liquid storage.
